lib/rescue_from_unless_local.rb

```diff
@@ -35,7 +37,12 @@ def rescue_from_unless_local(*klasses, with: nil, &block)
             end
           end
 
-        raise exception if Rails.application.config.consider_all_requests_local
+        if Rails.application.config.consider_all_requests_local
```

Comment:
if one is local and wants to test the app sending to Raven, this local check needs to be commented out. that's ok?

Reply:
No you can just set this flag in `development.rb` to the value you want or you can stub it in the specs (we do such stubbing in highlights)